src/util/mod.rs

```diff
@@ -0,0 +1,12 @@
+//! Self-contained, re-usable utilities that are not specific to this game.
+
+#![allow(dead_code)]
+#![allow(unused_imports)]
```

Comment:
Don't think this one's needed?

Reply:
Well I can remove these lints for now. IME these warnings tend to come up undesirably in `util/` code, but there won't be a warning if I remove them for this PR.

Reply:
Definitely agree for `dead_code`, but I'm curious about how `unused_imports` comes up.

Reply:
That's from re-exports.

Reply:
Added this back in, commented out and explained.